src/openfl/_internal/renderer/canvas/CanvasTextField.hx

```diff
@@ -91,25 +95,18 @@ class CanvasTextField {
 				
 				var transform = graphics.__renderTransform;
 				
-				if (renderSession.renderType == DOM) {
-					
-					var scale = CanvasRenderer.scale;
+				graphics.__canvas.width = Std.int (width * pixelRatio);
+				graphics.__canvas.height = Std.int (height * pixelRatio);
+				
+				context.setTransform (transform.a * pixelRatio, transform.b, transform.c, transform.d * pixelRatio, transform.tx * pixelRatio, transform.ty * pixelRatio);
```

Comment:
Move this below the `if (DisplayObject.__supportDOM && pixelRatio != 1) {` block to be more consistent to the old order of commands? Or does it make more sense like this?

And in `CanvasGraphics` we do similar stuff, but we have the `if (canvas.width == scaledWidth && canvas.height == scaledHeight) {` check to differ between a resized or not-resized canvas. I would not say we have to unify this now, but maybe it would make sense to do so. What do you think?